app/javascript/dashboard/components/ui/Dropdown/DropdownList.vue

```diff
@@ -1,9 +1,11 @@
 <script setup>
 import { ref, computed } from 'vue';
+import { debounce } from 'lodash';
```

Comment:
Why are we using `debounce` from `lodash` instead of `@chatwoot/utils`?

Reply:
Both are same. I have changed to `@chatwoot/utils`.